settings.gradle

```diff
@@ -14,6 +14,7 @@ if (JavaVersion.current().isJava8Compatible()) {
 }
 include 'libs:gretty-runner-tomcat'
 include 'libs:gretty-runner-tomcat7'
+include 'libs:gretty-runner-tomcat8and9'
```

Comment:
I don't suppose there's a way of doing this without locking that filename to Tomcat 8 and 9 - presumably we will need it for 10 also?  If Tomcat 7 is the odd-one-out, could we put this code into the base runner and make Tomcat 7 the exception?  Just 'supposin.

Reply:
Well, I'm not sure. There is this `gretty-runner-tomcat` project that is compiled against Tomcat 7. Then, the rest of `gretty-runner-tomcat#NUMBER` projects depend on the "main" one and add additional stuff on top of it. If we made the "main" one include the things from `8and9`, then `7` would not compile as it doesn't have the correct dependencies. Not sure if I explain it good enough. Or do you have some other idea?